pkg/kube-metrics/metrics.go

```diff
@@ -92,3 +93,20 @@ func generateMetricFamilies(kind string) []ksmetric.FamilyGenerator {
 		},
 	}
 }
+
+// GetNamespacesForMetrics wil return all namespaces which will be used to export the metrics
+func GetNamespacesForMetrics(operatorNs string) ([]string, error) {
+	ns := []string{operatorNs}
+
+	// Get the value from WATCH_NAMESPACES
+	watchNamespace, err := k8sutil.GetWatchNamespace()
+	if err != nil {
+		return nil, err
+	}
+
+	// Generate metrics from the WATCH_NAMESPACES value if it contains multiple namespaces
+	if strings.Contains(watchNamespace, ",") {
+		ns = strings.Split(watchNamespace, ",")
```

Comment:
Do we want to always export metrics in the `operatorNs` or is it intentional that we leave it out if specific watchNamespaces are set?

Reply:
If the WATCH_NAMESPACES are set wth the multi namespace scenario then, so far, it must contain the namespace where the operator is deployed already so because of this we can just replace the value.